### DISCUSSION TITLE
✅ WebSSH Pro purchased in 2013 and now it is freemium. Double paid??

### DISCUSSION THREAD

**Discussion by RavenSystem** [isontheline/pro.webssh.net#310] [Q&A] [answered]
Helo,

I purchased WebSSH Pro in 2013. I used it until it had not updates about 3 years. Now, I see that it is live again, but it is freemium with InAppPurchase. If I click on restore purchase, I get an error telling me that I have not purchases. Does it mean that I must purchase it AGAIN to access to all features?? I hope no.

Many thanks.

**Comment by isontheline** [comment, marked as answer]
Hello @RavenSystem 👋

No if you have downloaded or purchased WebSSH three years ago you don't need to buy it again.

Could you make me a screenshot of the bottom page of WebSSH settings? 
There are some versions I need to check.

Thanks

**Reply by RavenSystem** [reply]
Hi,

Some update: I check that I am able to add more than one connection. But family shares don't work.

Many thanks!

![IMG_0311](https://user-images.githubusercontent.com/8655274/120076712-4daa2600-c0a7-11eb-9e3d-e79e525c5071.jpeg)

**Reply by isontheline** [reply]
Never used family sharing option though.

If you need a promotional code for another device : please contact me by email here : https://mengus.net

Thanks for using WebSSH!

Have a nice day ☀️
